os/mm/asan/report.c

```diff
@@ -9,8 +9,9 @@
 #define SHADOW_BYTES_PER_BLOCK 1
 #define SHADOW_BLOCKS_PER_ROW 16
 #define SHADOW_BYTES_PER_ROW (SHADOW_BLOCKS_PER_ROW * SHADOW_BYTES_PER_BLOCK)
-#define SHADOW_ROWS_AROUND_ADDR 2
+#define SHADOW_ROWS_AROUND_ADDR 4
 
```

Comment:
Lot of code with __atribite__ compiler directives below make it gnu dependent. It cannot be a generic solution